ev3dev/brickpi.py

```diff
@@ -53,6 +53,7 @@ class Leds(object):
     LED1 = ( blue_led1, )
     LED2 = ( blue_led2, )
 
+    BLACK = ( 0, )
```

Comment:
So how should be go forward? 
1. Keep this as it is and merge?
1. Change BLACK to OFF and add an ON constant that is whatever is full brightness.